xl-v1-object_test.go

```diff
@@ -20,40 +20,13 @@ import (
 	"bytes"
 	"crypto/md5"
 	"encoding/hex"
-	"io/ioutil"
-	"os"
 	"testing"
 )
 
-func initializeObjLayer() (objLayer ObjectLayer, err error) {
-	initNSLock()
-	var disks []string
-	var d string
-	for i := 0; i < 8; i++ {
-		d, err = ioutil.TempDir("", "disk")
-		if err != nil {
-			return nil, err
-		}
-		disks = append(disks, d)
-	}
-	objLayer, err = newXLObjects(disks)
-	if err != nil {
-		return nil, err
-	}
-	return objLayer, nil
-}
-
-func cleanupObjLayer(xl xlObjects) {
-	disks := xl.physicalDisks
-	for i := range disks {
-		os.RemoveAll(disks[i])
-	}
-}
-
 func failDisks(xl xlObjects, n int) (removedDisks []StorageAPI) {
```

Comment:
Since the `erasureInfo.Distribution` randomizes the placement of data and parity blocks amongst the disks, it suffices to pick any `k` successive disks out of the total.